client/layout/index.jsx

```diff
@@ -165,7 +164,12 @@ class Layout extends Component {
 							sectionName={ this.props.sectionName }
 						/>
 					) }
-					<GlobalNotices id="notices" notices={ notices.list } />
+					<AsyncLoad
+						require="components/global-notices"
+						placeholder={ null }
```

Comment:
I'm starting to think `null` should be the default `placeholder`. :)

Reply:
Yes please 😉